crates/core/tests/snapshots/errors__error_display.snap

```diff
@@ -0,0 +1,41 @@
+---
+source: crates/core/tests/errors.rs
+expression: error
+---
+Well, this is embarrassing.
+
+`rustic_core` experienced an error related to `input/output operations`.
+
+Message:
+Prepended guidance line
+A file could not be read, make sure the file at `/path/to/file` is existing and readable by the system.
+Appended guidance line
+
+For more information, see: https://rustic.cli.rs/docs/errors/C001
+
+Related issues:
+- https://github.com/rustic-rs/rustic_core/issues/209
+
+We believe this is a bug, please report it by opening an issue at:
+https://github.com/rustic-rs/rustic_core/issues/new
+
+If you can, please attach an anonymized debug log to the issue.
+
+Thank you for helping us improve rustic!
+
+
+Some additional details ...
+
+Context:
+- path:	/path/to/file
+- called:	used s3 backend
```

Comment:
Probably not worth to have a `\t` here in between `key` and `value`. Should remove it, probably doesn't need to be aligned.